pypyr/cli.py

```diff
@@ -6,36 +6,63 @@
 import os
 import pypyr.pipelinerunner
 import pypyr.version
+import signal
+import sys
 
 
-def get_args():
+def get_args(args):
```

Comment:
I did look at click because of the juice I saw you use in cloco-cli - but I had a vague "avoid extra deps unless I really eally need them" sort of plan and pypyr actually on purpose has way simpler input args than cloco-cli - it's really just 3 or 4 flat args but thanks for reminder.